### ISSUE TITLE
Create training dataset with "Crop and Label Data": Filename appends extra '.' during generation.

### ISSUE THREAD

**Issue by catubc** [DeepLabCut/DeepLabCut#742]
Hi

I have now confirmed starting a new project from scratch that there is a filename bug in the above function.  

The DLC code in fact appends an extra '.' explicitly now, not sure why that is happening:

`    381                 video_orig = sep.join((vidpath, vidname + "." + videotype))`

**Desktop (please complete the following information about your system):**
 - Ubuntu 18.04
 - DeepLabCut Version: 2.26b

**To Reproduce**
Steps to reproduce the behavior, i.e.:
1. Label Frames
2. Run Create training data using the "Crop and Label Data" Option (this bug is not present if this is turned off.
```
(DLC-GPU) cat@cat-System-Product-Name:~/code/DeepLabCut$ ipython
^[[APython 3.7.6 (default, Jan  8 2020, 19:59:22) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.13.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import deeplabcut                                                                                                                                         
^[[A^[[A

In [2]: deeplabcut.launch_dlc()                                                                                                                                   
note to user: currently only ResNet50-v1 is available for maDLC
note to user: currently model comparison is not available in maDLC
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5/5 [00:12<00:00,  2.41s/it]
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/code/DeepLabCut/deeplabcut/gui/create_training_dataset.py in create_training_dataset(self, event)
    368                 ]
    369                 deeplabcut.cropimagesandlabels(
--> 370                     self.config, n_crops, (height, width), userfeedback
    371                 )
    372             else:

~/code/DeepLabCut/deeplabcut/generate_training_dataset/trainingsetmanipulation.py in cropimagesandlabels(config, numcrops, size, userfeedback, cropdata, excludealreadycropped, updatevideoentries)
    380                 # moving old entry to _original, dropping it from video_set and update crop parameters
    381                 video_orig = sep.join((vidpath, vidname + "." + videotype))
--> 382                 cfg["video_sets_original"][video_orig] = cfg["video_sets"][video_orig]
    383                 cfg["video_sets"].pop(video_orig)
    384                 cfg["video_sets"][video_orig.replace(vidname, new_vidname)] = {

~/anaconda3/envs/DLC-GPU/lib/python3.7/site-packages/ruamel/yaml/comments.py in __getitem__(self, key)
    751         # type: (Any) -> Any
    752         try:
--> 753             return ordereddict.__getitem__(self, key)
    754         except KeyError:
    755             for merged in getattr(self, merge_attrib, []):

KeyError: '/media/cat/4TBSSD/dlc_models/movies/2020-3-8_12_08_57_943006_compressed..avi'


```
</p></details>

**Comment by catubc** [comment]
[EDIT]: I removed the extra "." from the string concatenation and recompiled, the code works now.

**Comment by jeylau** [comment]
Hi @catubc, you have really made our lives easy with pointing exactly where the issue was. This will be fixed here #743 :)
